packages/ui/quantity/Quantity.tsx

```diff
@@ -0,0 +1,22 @@
+import { useCounter } from '@mantine/hooks';
+import { BsDash, BsPlus } from 'react-icons/bs';
+
+import { Count, QuantityWrapper, SquareButton } from './style';
+
+export function Quantity() {
+  const [count, handlers] = useCounter(0, { min: 0, max: 100 });
```

Comment:
as discussed, it won't be allowed to place 0 quantity. try changing the min value to "1" for this mantine hook.

Reply:
ok sir